jekyll/_cci1/parallel-manual-setup.md

```diff
@@ -16,11 +16,23 @@ Go to **Project Settings > Parallelism** to adjust the settings.
 
 When you use CircleCI's parallelization, we run your code on multiple separate VMs.
 To use parallelism, you make your test runner run only a subset of tests on each VM.
+
 There are two mechanisms for splitting tests among nodes: using the `files`
 configuration modifier (a very simple and straightforward way for most use cases), or
-using parallelism environment variables (aimed for  more complex scenarios).
+using parallelism environment variables (aimed at more complex scenarios).
+
+Either way, there are a few things to keep in mind:
+* `test` and `deploy` sections in your `circle.yml` are single-node only by
+  default and will only run on node 0. However, you can override `test` sections
+  per command (add `:` to the end of each command you'd like to parallelize, then
+  add a double-indented `parallel: true` below that). Alas, no parallel deploys.
+
+* Inferred `test` commands will automatically parallelize.
+
+* Currently, commands that only run in node 0 will erroneously show up in the
+  UI as running on all nodes.
 
-## Using the `files` configuration modifier
+<h3>Using the `files` configuration modifier</h3>
```

Comment:
You can change this to `###` for an H3.  Other places use an HTML tag if they need an HTML attribute.